procurement_purchase_requisition_generation/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+This module restores the purchase_requisition functionality of v15 in which purchas.requisition is generated instead of purchase.order.
```

Comment:
This is a user description, so better to avoid technical names

```suggestion
This module restores the v15 functionality in which you can configure products for generating purchase tenders instead of draft purchase orders.
```